packages/vendure-plugin-customer-managed-groups/src/api-extensions.ts

```diff
@@ -122,6 +152,15 @@ export class CustomerManagedGroupsResolver {
     return this.service.addToGroup(ctx, input);
   }
 
+  @Mutation()
+  @Allow(Permission.Authenticated)
+  async updateCustomerManagedGroupMember(
+    @Ctx() ctx: RequestContext,
+    @Args('input') input: UpdateCustomerManagedGroupMemberInput
+  ): Promise<CustomerManagedGroup | undefined> {
```

Comment:
We shouldnt return undefined here: if not in a group or not allowed we throw an error.